test/cpp/api/integration.cpp

```diff
@@ -14,6 +14,7 @@
 #include <test/cpp/api/util.h>
 
 using namespace torch::nn;
+using namespace torch::test;
```

Comment:
I made some small namespace changes here (stuff in `test/cpp/api/util.h` is now under `torch::test::`).